### ISSUE TITLE
DKIM Error

### ISSUE THREAD

**Issue by eakteam** [litespeedtech/cyberpanel#1176]
After generating DKIM and syncing to the cloudflare or adding manually from cyberpanel DNS it contains extra quotes and extra spaces. So it is not recognized as it should.

**Comment by usmannasir** [comment]
Delete DKIM related txt records from Cloudflare and let CyberPanel sync it, it should take care of this.

**Comment by eakteam** [comment]
Tried to do that many times. Still the same.

**Comment by usmannasir** [comment]
Can you open ticket with us and mention this issue, then team will be able to pass this onto me.

I will check if bug then fix.

Please provide all relevant login details via ticket like cloudflare, CyberPanel.

**Comment by shbs9** [comment]
@eakteam we need you server access can you please send us on platform to check

**Comment by eakteam** [comment]
@shbs9 you need it now? I can provide AnyDesk/TeamViewer support?

**Comment by shbs9** [comment]
can you first please open a ticket we have to debug this in deep

**Comment by eakteam** [comment]
@shbs9 @usmannasir https://platform.cyberpersons.com/support/KSGB8TQVB

**Comment by usmannasir** [comment]
@eakteam take a look here -> https://community.cyberpanel.net/t/message-not-signed-with-dkim/50594/18

This thread could solve your issue.

**Comment by eakteam** [comment]
I will try for sure. It looks like the same issue...

**Comment by usmannasir** [comment]
It worked for you?

**Comment by eakteam** [comment]
@usmannasir the file /etc/opendkim.conf is exactly as on your post :
```

Syslog			yes
SyslogSuccess		yes

Canonicalization	relaxed/simple
OversignHeaders		From


UserID			opendkim
UMask			007

Socket  inet:8891@localhost

PidFile			/run/opendkim/opendkim.pid


TrustAnchorFile		/usr/share/dns/root.key

Mode	sv
Canonicalization	relaxed/simple
KeyTable	refile:/etc/opendkim/KeyTable
SigningTable	refile:/etc/opendkim/SigningTable
ExternalIgnoreList	refile:/etc/opendkim/TrustedHosts
InternalHosts	refile:/etc/opendkim/TrustedHosts
```

But emails still shows as not signed by DKIM
I have replied also to that thread https://community.cyberpanel.net/t/message-not-signed-with-dkim/50594/18

**Comment by usmannasir** [comment]
fixed at code level in v2.3.5-dev
